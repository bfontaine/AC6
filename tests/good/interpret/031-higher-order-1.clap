def apply (f) (x) = f (x)
val z = apply (fun (x) => x + 1) 41

