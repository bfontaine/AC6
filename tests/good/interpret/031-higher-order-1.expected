:- apply = <code>
:- z = 42
